pkg/preflight/checks.go

```diff
@@ -273,7 +273,7 @@ func (dac DirExistingCheck) Check() (warnings, errorList []error) {
 	defer f.Close()
 
 	_, err = f.Readdirnames(1)
-	if err == io.EOF {
+	if errors.Is(err, io.EOF) {
```

Comment:
There are some introduction about `io.EOF` In the link: https://github.com/golang/go/wiki/ErrorValueFAQ
> Comparisons to io.EOF need not be changed, because io.EOF should never be wrapped.

so i think maybe we should keep the `if err == io.EOF` comparison code. how about recover all of modifications about `io.EOF` comparisons?

Reply:
ok, i will fix all